ansible/roles/providers/tasks/vmware.yml

```diff
@@ -29,6 +29,8 @@
   yum:
     name: "{{ packages }}"
     state: present
+    enablerepo: "{{ 'offline' if offline_mode_enabled else '' }}"
+    disablerepo: "{{ '*' if offline_mode_enabled else '' }}"
   vars:
     packages:
       - cloud-init
```

Comment:
I also see that these packages are already included in our air-gapped artifacts https://github.com/mesosphere/dkp-release/blob/1d49a12fb7147e7b4863f14043e9806ee279bd95/os-packages-bundles/Makefile#L47

Reply:
Thanks for checking.